### ISSUE TITLE
[2019.09] PHP Notices

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#7299]
Please report here any PHP Notice message obtained running the `2019.06` stable or `2019.09-dev` version.

**Comment by copiis** [comment]
`PHP Notice:  Undefined index: nickname in /friendica/addon/twitter/twitter.php on line 1887`

**Comment by copiis** [comment]
`PHP Notice:  Uninitialized string offset: 1 in /friendica/src/Model/Item.php on line 2501`

Im on Stable

**Comment by AlfredSK** [comment]
2019.09-dev:
```
[30-Jun-2019 15:17:22 UTC] PHP Notice:  Undefined offset: 0 in .../src/Util/Crypto.php on line 213
[30-Jun-2019 15:17:22 UTC] PHP Notice:  Trying to get property 'asnData' of non-object in .../src/Util/Crypto.php on line 213
```

**Comment by AlfredSK** [comment]
2019.06-dev:

```
Undefined index: network in /var/www/html/libranet/src/Model/Contact.php on line 1372
Undefined index: network in /var/www/html/libranet/src/Model/Contact.php on line 1400
```

**Comment by AlfredSK** [comment]
2019.09-dev

`Undefined property: Friendica\Core\L10n\L10n::$langSave in .../src/Core/L10n/L10n.php on line 123`

**Comment by MrPetovan** [comment]
Paging @nupplaphil

**Comment by copiis** [comment]
2019.06-stable
PHP Notice: Undefined index: nickname in /friendica/addon/twitter/twitter.php on line 1887

**Comment by copiis** [comment]
2019.06-stable
`PHP Notice:  Undefined offset: 401 in /friendica/addon/buffer/bufferapp.php on line 134`

**Comment by copiis** [comment]
Develop
`
PHP Notice:  Undefined index: url in /addon/twitter/twitter.php on line 624`

**Comment by copiis** [comment]
```
PHP Notice:  Undefined index: id in /src/Worker/Delivery.php on line 359
PHP Notice:  Undefined index: id in /src/Worker/Delivery.php on line 413
```
develop

**Comment by AlfredSK** [comment]
Use the source, Luke!

`Undefined index: guid in .../src/Module/Admin/Item/Source.php on line 27`

**Comment by copiis** [comment]
```
PHP Notice:  Undefined offset: 0 in /src/Model/GContact.php on line 758
PHP Notice:  Undefined offset: 0 in /src/Model/GContact.php on line 787
PHP Notice:  Undefined offset: 0 in /src/Model/GContact.php on line 822
PHP Notice:  Undefined offset: 0 in /src/Model/GContact.php on line 824
PHP Notice:  Undefined offset: 0 in /src/Model/GContact.php on line 830
```

**Comment by AlfredSK** [comment]
RC:

```
PHP Notice:  Undefined variable: a in .../src/Module/Bookmarklet.php on line 45
PHP Notice:  Trying to get property 'user' of non-object in .../src/Module/Bookmarklet.php on line 45
```

**Comment by AlfredSK** [comment]
RC:

```
PHP Notice:  Undefined property: stdClass::$screen_name in .../addon/twitter/twitter.php on line 356
PHP Notice:  Undefined property: stdClass::$screen_name in .../addon/twitter/twitter.php on line 360
PHP Notice:  Undefined property: stdClass::$profile_image_url in .../addon/twitter/twitter.php on line 360
PHP Notice:  Undefined property: stdClass::$description in .../addon/twitter/twitter.php on line 361
```

**Comment by copiis** [comment]
RC
`PHP Notice:  Undefined variable: return in /src/Core/ACL.php on line 363\n', referer: https://meld.de/contact/261757`

**Comment by AlfredSK** [comment]
Some 'undefined index' notices related to events:

```
[14-Sep-2019 06:45:09 UTC] PHP Notice:  Undefined index: summary in .../mod/events.php on line 435
[14-Sep-2019 06:45:09 UTC] PHP Notice:  Undefined index: desc in .../mod/events.php on line 436
[14-Sep-2019 06:45:09 UTC] PHP Notice:  Undefined index: location in .../mod/events.php on line 437
[14-Sep-2019 06:45:09 UTC] PHP Notice:  Undefined index: id in .../mod/events.php on line 438
[14-Sep-2019 06:45:09 UTC] PHP Notice:  Undefined index: cid in .../mod/events.php on line 439
[14-Sep-2019 06:45:09 UTC] PHP Notice:  Undefined index: uri in .../mod/events.php on line 440
[14-Sep-2019 06:45:09 UTC] PHP Notice:  Undefined index: allow_cid in .../mod/events.php on line 446
[14-Sep-2019 06:45:09 UTC] PHP Notice:  Undefined index: finish in .../mod/events.php on line 459
[14-Sep-2019 06:45:09 UTC] PHP Notice:  Undefined index: adjust in .../mod/events.php on line 463
```

**Comment by copiis** [comment]
RC
`PHP Notice:  Uninitialized string offset: 1 in /src/Model/Item.php on line 2536`

**Comment by AlfredSK** [comment]
RC: 

```
[15-Sep-2019 10:00:25 Europe/Berlin] PHP Notice:  Undefined index: dob in .../mod/profiles.php on line 210
[15-Sep-2019 10:00:25 Europe/Berlin] PHP Notice:  Undefined index: gender in .../mod/profiles.php on line 242
[15-Sep-2019 10:00:25 Europe/Berlin] PHP Notice:  Undefined index: detailed_profile in .../mod/profiles.php on line 341
```

**Comment by copiis** [comment]
RC
`PHP Notice:  Undefined index: url in /addon/twitter/twitter.php on line 633`

**Comment by AlfredSK** [comment]
[tumblr] on RC:

```
PHP Notice:  Undefined index: tumblr_bydefault in .../addon/tumblr/tumblr.php on line 302
PHP Notice:  Undefined index: tumblr in .../addon/tumblr/tumblr.php on line 300
```

**Comment by nupplaphil** [comment]
Closed in favor of #7677
